tools/DaemonArchitecture/qprocessordetection.h

```diff
@@ -0,0 +1,390 @@
+// Copyright (C) 2016 The Qt Company Ltd.
```

Comment:
In the commit message, please say which version/revision of Qt the file comes from.

Reply:
Done.